covid19-algorithme-orientation-openapi.json

```diff
@@ -245,14 +238,13 @@
             },
             "risk_factors": {
               "breathing_disease": true,
-              "cancer": true,
```

Comment:
Why the "cancer" line has been removed here?

Reply:
oups !